buildSrc/src/test/groovy/org/elasticsearch/gradle/doc/SnippetsTaskTests.groovy

```diff
@@ -0,0 +1,43 @@
+package org.elasticsearch.gradle.doc
+
+import org.elasticsearch.gradle.test.GradleUnitTestCase
+
+class SnippetsTaskTests extends GradleUnitTestCase {
```

Comment:
Please add a Java test. We are looking to replace groovy all new classes should be Java.

Reply:
Done